doc/ap_verify_hits2015/index.rst

```diff
@@ -0,0 +1,40 @@
+.. _ap_verify_hits2015-package:
+
+##################
+ap_verify_hits2015
+##################
+
+The ``ap_verify_hits2015`` package represents a small subset of the DECam `HiTS <http://astroinf.cmm.uchile.cl/the-high-cadence-transient-survey/>`_ survey.
+It is a good example dataset for difference imaging with DECam.
+
+Project info
+============
+
+Repository
+   https://github.com/lsst/ap_verify_hits2015
+
+.. Datasets do not have their own (or a collective) Jira components; by convention we include them in ap_verify
+
+Jira component
+   `ap_verify <https://jira.lsstcorp.org/issues/?jql=project %3D DM %20AND%20 component %3D ap_verify %20AND%20 text ~ "hits2015">`_
+
+Dataset Contents
+================
+
+This package provides complete data for three fields from the 2015 HiTS survey.
+It contains:
+
+* raw images from the ``Blind15A_26``, ``Blind15A_40``, and ``Blind15A_42`` survey fields, in g band.
+* biases (``zci``), flats (``fci``), and corresponding weight maps (``zcw``, ``fcw``) in all DECam bands (ugriz, Y, VR). [Note: weights are not yet supported by the Stack]
```

Comment:
Reword to avoid implying things about future Stack plans, and because weight maps are somewhat supported if you are using Instcals rather than raws. Something like "Note: DECam weight maps are not used by ap_verify or ap_pipe." Same for your illumcor comment.